Podfile

```diff
@@ -14,12 +20,13 @@ end
 target "ReactiveFeedbackTests" do
     platform :ios, "8.0"
     shared_pods
-    pod "Nimble", "~> 7.0"
+    pod "Nimble", "~> 8.0"
 end
 
-target "Example" do
-    platform :ios, "10.0"
-    shared_pods
-    pod "Kingfisher", "~> 4.0"
-    pod "ReactiveCocoa", "~> 8.0"
-end
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['SWIFT_VERSION'] = '4.2'
```

Comment:
Is this because dependent pods don't have the swift version set correctly?

Reply:
This need to be reviewed at a later PR.